### PR TITLE
Hide collection duration if 0

### LINKED ISSUE #2278: [Bug]: Collection view shows ebook duration in seconds
### Describe the issue

The collection view shows a duration in seconds for ebooks which doesn't really make sense as it's always just 0, since ebooks don't have a duration.
It would be better to show the read percentage (or no value at all).

image

### Steps to reproduce the issue

1. Add ebook to collection
2. View collection

### Audiobookshelf version

v2.5.0

### How are you running audiobookshelf?

Docker

### PR DESCRIPTION
**Pull request by brianjaustin**
This fixes #2278 by hiding the duration of a book in a collection if it is 0 (or less). I suppose some audiobooks could also have a duration of 0, but this seems rare (and perhaps hiding the duration in those cases also makes sense?). I opted not to go with progress as that would result in the value having different meanings depending on the media type, which sounds like a potentially confusing user experience.

While this fixes the reported bug, I wonder if it makes sense to have page count as the "duration". Although that would require more effort to implement I think.

### CODE REVIEW

**Comment by nichwall** [comment]
The only problem I see with using "pages" as duration is that only works for PDF or comics (epub page count is relative based on zoom and screen size).

I can't think of any reason to not hide duration when it's zero. That indicates there are either no audio items, or all of the audio items are zero duration. When people make placeholder books, they tend to use empty files, so there still wouldn't be a duration.

**Comment by advplyr** [comment]
Yeah as @nichwall said, we can't get number of pages for epub. I was thinking of showing the ebook format in-place of the duration

File: client/components/tables/collection/BookTableRow.vue [hunk 1]
```diff
@@ -30,7 +30,7 @@
               ><span :key="author.id + '-comma'" v-if="index < bookAuthors.length - 1">,&nbsp;</span>
             </template>
           </div>
-          <p class="text-xs md:text-sm text-gray-400">{{ bookDuration }}</p>
+          <p class="text-xs md:text-sm text-gray-400" v-if="media.duration > 0">{{ bookDuration }}</p>
```
Comment (advplyr) [inline client/components/tables/collection/BookTableRow.vue:33]:
```suggestion
          <p v-if="media.duration" class="text-xs md:text-sm text-gray-400">{{ bookDuration }}</p>
```

**Comment by advplyr** [comment]
Thanks!

**Pull request merged**